app/src/main/java/com/example/trainingintive/dog_images_feature/data/local/DogImageDao.kt

```diff
@@ -16,4 +18,10 @@ interface DogImageDao {
 
     @Query("SELECT * from dogImageUrl")
     fun getAllImageUrls(): Flowable<List<DogImageUrl>>
+
+    @Update
+    fun update(imageUrl: DogImageUrl): Completable
```

Comment:
You should not save the domain layer objects in the local database because it is the data layer. You can create a separate objects for storing in the database and then map to/from the domain